Overview.src.html

```diff
@@ -3026,6 +3092,46 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  to `<code title>GET</code>` and <var>request</var>'s <span title=concept-request-body>body</span>
  to null.
 
+ <li>
+   <p>Let <var>useReferredTokenBinding</var> be the result of
+   <span title=concept-header-parse>parsing</span>
+   `<code title>Include-Referred-Token-Binding-ID</code>` in <var>actualResponse</var>'s
+   <span title=concept-response-header-list>header list</span>.
+   The value of this <span title=concept-header>header</span> is specified in
+   <a href="https://tools.ietf.org/html/draft-ietf-tokbind-https#section-4.3">section 4.3</a>
+   of the Token Binding over HTTP spec.
+   <p class="note no-backref">By setting the <code title>Include-Referred-Token-Binding-ID</code>
+   <span title=concept-header>header</span> to "<code title>true</code>",
+   the <span title=concept-request-origin>origin</span> that sends the
+   redirect <span title=concept-response>response</span> tells the user agent to disclose the
+   <span title=concept-token-binding-id>Token Binding Id</span> used by the user agent with that
+   <span title=concept-request-origin>origin</span> to the target origin.
+
+   <li><p>Unset the <var>request</var>'s
+     <span title=concept-request-use-referred-token-binding>use-referred-token-binding flag</span>.
+     <li><p>If <var>useReferredTokenBinding</var> is a <span>byte-case-insensitive</span>
+     match for `<code title>true</code>`, and the <var>request</var>'s
+     <span title=concept-request-use-token-binding>use-token-binding flag</span> is set,
+     then perform the following substeps:
+
+   <ol>
+     <li><p>Set the <var>request</var>'s
+       <span title=concept-request-use-referred-token-binding>use-referred-token-binding</span>
+       flag.
+       <li><p>Set the <var>request</var>'s
+         <span title=concept-request-referred-token-binding-origin>referred-token-binding-origin</span>
+         to <var>request</var>'s <span title=concept-request-origin>origin</span>.
+   </ol>
+
+   <p class="note no-backref">By causing the
+   <span title=concept-request-use-referred-token-binding>use-referred-token-binding flag</span>
+   to be set for the subsequent <span title=concept-request>request</span> via the
+   <code title>Include-Referred-Token-Binding-ID</code> <span title=concept-header>header</span>
+   in its <span title=concept-response>response</span>, the referring
+   <span title=concept-request-origin>origin</span> tells the user agent to disclose to the target
+   server the <span title=concept-token-binding-id>Token Binding Id</span> used by the user agent
+   with the referring <span title=concept-request-origin>origin</span>.
```

Comment:
It's probably worth moving all of this (notes included) out into an algorithm in the token binding section to simplify HTTP-redirect fetch, which is already complicated enough. :)